### ISSUE TITLE
Related categories from a category query

### ISSUE THREAD

**Issue by membla** [markhuot/craftql#177]
I have a main category that itself contains a field for sub categories. The two categories belong to different category groups. Can I use craftql to create a query that exposes all main categories, within them their sub categories, and within these the entries that are related to the respective sub category?

**Comment by markhuot** [comment]
This is now possible. All category fields applied to entries expose two fields, the name of the field as well as the name of the field + `Connection`. E.g.,

```graphql
{
  entriesConnection(limit:10) { # pull 10 entries
    edges {
      node {
        title # get the title of the 10 entries
        categoryFieldHandleConnection { # get the categories of on each of the 10 entries
          edges {
            node {
              title # get the title of the related category
            }
            relatedEntries { # get related entries to each category applied to the 10 entries
              entries {
                title
              }
            }
          }
        }
      }
    }
  }
}
```

**Comment by markhuot** [comment]
Closing this issue. The fix is included in version 1.2.1.
